test/erlfmt_SUITE.erl

```diff
@@ -1147,11 +1230,29 @@ snapshot_enclosing_range_no_leak(_) ->
         "y()   ->  1.\n",
     Reference =
         "x() ->\n"
-        "    0.",
+        "    0.\n"
+        "\n"
+        "y()   ->  1.\n",
     % End point overshots line 3, but doesn't reach line 4:
     % Only x() must be formatted.
-    Output = erlfmt:format_string_range(Original, {1, 1}, {3, 3}, []),
-    assert_snapshot_match(list_to_binary(Reference), Output).
+    Result = erlfmt:format_string_range(Original, {1, 1}, {3, 3}, []),
+    assert_snapshot_match_range(Reference, Result).
+
+snapshot_range_reinjected(_) ->
+    % Verify the formatted range is properly re-injected
+    % in original file, even if the number of lines has changed.
+    Original =
+        "x()->0. % This comment end up in new line.\n"
```

Comment:
Maybe we do want a few more tests where the number of lines are becoming more, like in this one.  But also where it is not the first form.
But I think it is okay to add that after merging this.

Reply:
I also think it is okay :).  This version is fit to test IDE integration!